packages/expo-ui/src/index.ts

```diff
@@ -1,8 +1,13 @@
-export type { Switch, SwitchProps } from '../components/Switch';
-export type { Picker, PickerProps } from '../components/Picker';
-export type { Button, ButtonProps } from '../components/Button';
-export type { Section, SectionProps } from '../components/Section';
-export type { Slider, SliderProps } from '../components/Slider';
+export type { SwitchProps } from './components/Switch';
+export type { PickerProps } from './components/Picker';
+export type { ButtonProps } from './components/Button';
+export type { SectionProps } from './components/Section';
+export type { SliderProps } from './components/Slider';
+export { Switch } from './components/Switch';
```

Comment:
this was a type export for a purpose – we don't want to use the [barrel file](https://dev.to/thepassle/a-practical-guide-against-barrel-files-for-library-authors-118c)as our main export since it will cause all components to be imported if even one is imported.

Let's keep the component name in the import statement
```
import { Picker } from '@expo/ui/Picker';
```
If there's a clean way to get both node and TS to remap this path to `@expo/ui/src/Picker/index.ts` than I'm fine with moving all files to `src`, but I didn't find it when creating the package 😞 

What's the main reason for moving everything to `src`?

Reply:
For me, the imports didn't work at all without this change, or something like it. Thanks for the explanation -- I'll revert these changes and see if there is a way to have the components imported separately.

Reply:
Changes are reverted.

I am not sure why there is a problem with importing all components -- the JS side of this package doesn't seem that large, so the bundle size shouldn't increase too much. However, if that is really necessary, I think a cleaner solution would be to have a single `@expo-ui/core` with all the native code, and individual very small JS-only packages like `@expo-ui/slider`, `@expo-ui/switch`, etc.

I'll give this some more thought and try out some ideas. In the meantime, let's go ahead and just merge the tvOS support here.